fix: typo in restoreRelease

jinqi7 opened the pull request:
The name of the release file was  added a wrong prefix.  It's to change it to the correct one.

# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Issue ticket number and link

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added meaningful description with JIRA/GitHub issue key(if applicable), for example HASSuiteDescribe("STONE-123456789 devfile source") 
- [ ] I have updated labels (if needed)

jinqi7 commented:
/retest

jinqi7 commented:
/retest

theflockers reviewed (commented):
the code look ok, but I am afraid your PR would mess up a bit with the commit tree. Not sure exactly how to fix it . Any idea @johnbieren @happybhati

johnbieren commented:
> the code look ok, but I am afraid your PR would mess up a bit with the commit tree. Not sure exactly how to fix it . Any idea @johnbieren @happybhati

I'm not really sure what you are saying about the commit tree. This should just be rebased and squashed down to one commit, is that what you mean? I also don't get what this PR is doing. @jinqi7 it would be really nice to include the why of changes you are making in the commit messages

theflockers commented:
> > the code look ok, but I am afraid your PR would mess up a bit with the commit tree. Not sure exactly how to fix it . Any idea @johnbieren @happybhati
> 
> I'm not really sure what you are saying about the commit tree. This should just be rebased and squashed down to one commit, is that what you mean? I also don't get what this PR is doing. @jinqi7 it would be really nice to include the why of changes you are making in the commit messages

I was unsure if only squashing would fix it. thansk @johnbieren

jinqi7 commented:
> > > the code look ok, but I am afraid your PR would mess up a bit with the commit tree. Not sure exactly how to fix it . Any idea @johnbieren @happybhati
> > 
> > 
> > I'm not really sure what you are saying about the commit tree. This should just be rebased and squashed down to one commit, is that what you mean? I also don't get what this PR is doing. @jinqi7 it would be really nice to include the why of changes you are making in the commit messages
> 
> I was unsure if only squashing would fix it. thansk @johnbieren

It's my bad. I rebased my PR and I only want to submit one commit.

psturc reviewed (approved):
/lgtm

jinqi7 commented:
/retest

johnbieren reviewed (approved):
/approve
/lgtm

jinqi7 commented:
/retest

jinqi7 commented:
/retest

jinqi7 commented:
/retest
